tests/e2e/utils/flows.js

```diff
@@ -397,6 +397,11 @@ export const merchantWCP = {
 	},
 
 	addNewPageCheckoutWCB: async () => {
+		// Set a User Agent different than the "puppeteer-debug wp-e2e-test" one we use
+		// so the "Add block" button isn't disabled during -dev tests.
+		page.setUserAgent(
```

Comment:
@deepakpathania I didn't find a sample User-Agent that didn't describe an OS/browser that worked for the tests. Glad to go with a different approach.

Reply:
I feel we could maybe move this to `jest-setup.js`, create a function for it and add that to the `beforeAll` block so that it applies to all test suite runs since this could be applicable for other tests as well. WDYT?

Reply:
Here as well, I'd say to get a second opinion from someone who's more familiar with the tests than I am if you'd like to discuss that approach 🙂 

I think having it before the redirect to the blocks editor would be of less impact, I'm not sure about the consequences that having that User-Agent instead of having the default `puppeteer-debug wp-e2e-test` value could have in other tests. Looks like that's set in our [puppeteer-utils library](https://github.com/Automattic/puppeteer-utils/blob/master/src/setup-env.js#L16). Also, other tests are running fine with the default value, right? So, yep, not sure which path would be better. Let me know if we'd like to go that way and I can update it.

Reply:
Discussing here: p1638884905038500-slack-C026J0AL8NS

Reply:
Aand User-Agent updated for other tests in [0497e38](https://github.com/Automattic/woocommerce-payments/pull/3277/commits/0497e388b63146b1410826b2b93c668ebe868f3e).

I tried using the `beforeAll` for setting this but the User-Agent wasn't being updated for some reason. I tried this when we first started discussing this so I don't recall the details. I can give it another try if having it under `beforeAll` would be better than the current approach.